storybook/stories/components/CheckRequirements.stories.js

```diff
@@ -37,28 +37,71 @@ export const Default = {
     requirements: {
       requirements_ok: true,
       errors: [
-        'Your current PHP version isn\'t compatible with PrestaShop 9.0.0. (Expected: 8.1 - 8.3 | Current: 8.0)',
-        'Your store\'s root directory xxx isn\'t writable. Provide write access to the user running PHP with appropriate permission & ownership.',
-        'The "/admin/autoupgrade" directory isn\'t writable. Provide write access to the user running PHP with appropriate permission & ownership.',
-        'PHP\'s "Safe mode" needs to be disabled.',
-        'Files can\'t be downloaded. Enable PHP\'s "allow_url_fopen" option or install PHP extension "cURL".',
-        'Missing PHP extension "zip".',
-        'Maintenance mode needs to be enabled. Enable maintenance mode and add your maintenance IP in Shop parameters > General > Maintenance.',
-        'PrestaShop\'s caching features needs to be disabled. Disable caching features in Advanced parameters > Performance > Caching',
-        'PHP\'s max_execution_time setting needs to have a high value or needs to be disabled entirely (current value: 15 seconds)',
-        'Apache mod_rewrite needs to be enabled.',
-        'The following PHP extensions needs to be installed: curl, dom, fileinfo, gd',
-        'The following PHP functions needs to be allowed: fopen, fclose, fread, fwrite',
-        'PHP memory_limit needs to be greater than 256 MB.',
-        'PHP file_uploads configuration needs to be enabled.',
-        'Unable to generate private keys using openssl_pkey_new. Check your OpenSSL configuration, especially the path to openssl.cafile.',
-        'It\'s not possible to write in the following folders: xxx, yyy, zzz. Provide write access to the user running PHP with appropriate permission & ownership.',
-        'The version of PrestaShop does not match the one stored in database. Your database structure may not be up-to-date and/or the value of PS_VERSION_DB needs to be updated in the configuration table.',
+        {
+          message: 'Your current PHP version isn\'t compatible with PrestaShop 9.0.0. (Expected: 8.1 - 8.3 | Current: 8.0)'
+        },
+        {
+          message: 'Your store\'s root directory xxx isn\'t writable. Provide write access to the user running PHP with appropriate permission & ownership.'
+        },
+        {
+          message: 'The "/admin/autoupgrade" directory isn\'t writable. Provide write access to the user running PHP with appropriate permission & ownership.'
+        },
+        {
+          message: 'PHP\'s "Safe mode" needs to be disabled.'
+        },
+        {
+          message: 'Files can\'t be downloaded. Enable PHP\'s "allow_url_fopen" option or install PHP extension "cURL".'
+        },
+        {
+          message: 'Missing PHP extension "zip".'
+        },
+        {
+          message: 'Maintenance mode needs to be enabled. Enable maintenance mode and add your maintenance IP in Shop parameters > General > Maintenance.'
```

Comment:
Looks like we can't check your changes on storybook because the text is sent without links

Reply:
```suggestion
          message: 'Maintenance mode needs to be enabled. Enable maintenance mode and add your maintenance IP in <a class="link">Shop parameters > General > Maintenance</a>.'
```